docs/pages/web/buttontoggle.tsx

```diff
@@ -0,0 +1,321 @@
+import LocalizationSection from '../../docs-components/LocalizationSection';
+import MainSection from '../../docs-components/MainSection';
+import Page from '../../docs-components/Page';
+import PageHeader from '../../docs-components/PageHeader';
+import QualityChecklist from '../../docs-components/QualityChecklist';
+import SandpackExample from '../../docs-components/SandpackExample';
+import changeLabelsDo from '../../examples/buttontoggle/changeLabelsDo';
+import colors from '../../examples/buttontoggle/colors';
+import confirmationDo from '../../examples/buttontoggle/confirmationDo';
+import consistencyDo from '../../examples/buttontoggle/consistencyDo';
+import defaultStateExample from '../../examples/buttontoggle/defaultStateExample';
+import disabledStateExample from '../../examples/buttontoggle/disabledStateExample';
+import iconEndExample from '../../examples/buttontoggle/iconEndExample';
+import localization from '../../examples/buttontoggle/localization';
+import locationDo from '../../examples/buttontoggle/locationDo';
+import main from '../../examples/buttontoggle/main';
+import selectedStateExample from '../../examples/buttontoggle/selectedStateExample';
+
+const PREVIEW_HEIGHT = 300;
+
+export default function DocsPage({ generatedDocGen }: DocType) {
+  return (
+    <Page title={generatedDocGen?.displayName}>
+      <PageHeader description={generatedDocGen?.description} name={generatedDocGen?.displayName}>
+        <SandpackExample
+          code={main}
+          hideEditor
+          name="Main ButtonToggle example"
+          previewHeight={150}
+        />
+      </PageHeader>
+
+      <GeneratedPropTable generatedDocGen={generatedDocGen} />
+
+      <MainSection name="Usage guidelines">
+        <MainSection.Subsection columns={2}>
+          <MainSection.Card
+            cardSize="md"
+            description={`
+            - The ButtonToggle should be used when you require a binary component with distinct on/off states
+            - To demonstrate that items are actively selected for filtering
+        `}
+            title="When to use"
+            type="do"
+          />
+          <MainSection.Card
+            cardSize="md"
+            description={`
+            - Avoid using the ButtonToggle when a simpler Checkbox, RadioButton, or Switch can be used instead.
+            - Do not use ButtonToggle in replacement of a Button, it should only be used for selected and unselected functionality
+        `}
+            title="When not to use"
+            type="don't"
+          />
+        </MainSection.Subsection>
+      </MainSection>
+      <MainSection name="Best practices">
+        <MainSection.Subsection columns={2}>
+          <MainSection.Card
+            cardSize="md"
+            description={`To make it clearer, you may want to change the label text to indicate that the ButtonToggle has been selected. For instance, changing "follow" to "Following."`}
```

Comment:
"follow" to "Following >> "**F**ollow" to "Following.

Reply:
DONE